ios/OpenTokReactNative/OTSessionManager.swift

```diff
@@ -399,12 +399,25 @@ class OTSessionManager: RCTEventEmitter {
         resolve(supportedCodecs)
     }
     
-    @objc func forceMuteAll(_ sessionId: String, excludedStreamIds: NSArray, resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) -> Void{
+    @objc func forceMuteAll(_ sessionId: String, excludedStreamIds: Array<String>, resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) -> Void{
         guard let session = OTRN.sharedState.sessions[sessionId] else {
             reject("event_failure", "Session ID not found", nil)
             return
         }
-        return resolve(true);
+        var excludedStreams:[OTStream] = []
```

Comment:
Hi Jeff, the similar function in Android code (https://github.com/opentok/opentok-react-native/blob/9dc34355aaf04950af8e32f78ea7b48e9fef6b06/android/src/main/java/com/opentokreactnative/OTSessionManager.java#L322) checks for multiple sessions; Is it required? 
Also the java function returns immediately after a stream is not found for streamID.  May be it can continue and loop through the valid ones. (Just happened to notice this randomly.)

Reply:
1. The java code you cite only check a single session:

```
ConcurrentHashMap<String, Session> mSessions = sharedState.getSessions();
Session mSession = mSessions.get(sessionId);
```

It could be refactored into one line, but this pattern is used in other code.

2. Good catch on the java function returning. I added a commit to fix this.

Reply:
> 1. The java code you cite only checks a single session (It could be refactored into one line, but this pattern is used in other code):
> 
> ```
> ConcurrentHashMap<String, Session> mSessions = sharedState.getSessions();
> Session mSession = mSessions.get(sessionId);
> ```
> 
> 2. Good catch on the java function returning. I added a commit to fix this.

For 1. ConcurrentHashMap<String, Session> mSessions = sharedState.getSessions();
My concern is whether we are expecting more than one session to exist at same time? (sharedState.getSession**s**())